### PR TITLE
[documentation] re-structure table of contents 

### PR DESCRIPTION
**Pull request by Yoyozilla**
This is in response to some of the feedback we have gotten.
The documents today are more like feature description and overview vs how-to.
How-to should be scenario/task focused to teach people how to complete a tast/scenario.
I am moving the doc ToC reflect more of this.

### CODE REVIEW

**Comment by wiwei** [comment]
Waiting on docs run, changes seem okay.

**Comment by wiwei** [comment]
Can you fix the doc build breaks please:


Build failed.
[19-04-24 10:10:29.536]Warning:[BuildCommand.BuildCore.Build Document.CompilePhaseHandlerWithIncremental.TocDocumentProcessor.Prebuild.BuildTocDocument](toc.yml)Unable to find either toc.yml or toc.md inside Documentation/. Make sure the file is included in config file docfx.json!
[19-04-24 10:10:29.536]Error:[BuildCommand.BuildCore.Build Document.Load.TocDocumentProcessor](Documentation/toc.yml)D:\a\1\s\Documentation\toc.yml is not a valid TOC File: D:\a\1\s\Documentation\toc.yml is not a valid TOC file, detail: (Line: 11, Col: 5, Idx: 212) - (Line: 11, Col: 5, Idx: 212): While parsing a block collection, did not find expected '-' indicator..
[19-04-24 10:10:29.536]Error:[BuildCommand.BuildCore.Build Document.Load.TocDocumentProcessor](Documentation/toc.yml)Unable to load file 'Documentation/toc.yml' via processor 'TocDocumentProcessor': D:\a\1\s\Documentation\toc.yml is not a valid TOC File: D:\a\1\s\Documentation\toc.yml is not a valid TOC file, detail: (Line: 11, Col: 5, Idx: 212) - (Line: 11, Col: 5, Idx: 212): While parsing a block collection, did not find expected '-' indicator..
	1 Warning(s)

**Comment by julenka** [comment]
@Yoyozilla  Sorry, but what is this restructuring exactly? Is it the table of contents for github.io page? Could you provide a little bit more detail in the pull request please?

**Comment by Yoyozilla** [comment]
This is in response to some of the feedback we have gotten. 
The documents today are more like feature description and overview vs how-to. 
How-to should be scenario/task focused to teach people how to complete a tast/scenario. 
I am moving the doc ToC reflect more of this.

**Comment by julenka** [comment]
Hi yoyo, it looks like the docs build is still failing. The error is: 

(Documentation/toc.yml#L0) **Invalid file link:(~/Documentation/README.md).**

**Comment by wiwei** [comment]
I think also the general feedback was to have in the PR description what is being changed and why. (If you could take your response to Julia's feedback and paste it into the PR description that would be good)

**Comment by wiwei** [comment]
Per our offline chat, I deleted the stale migration readme, but this causes a merge conflict in yours because I had to update the toc.yml

https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/4090

File: Documentation/toc.yml [hunk 1]
```diff
@@ -97,20 +99,32 @@
...
       href: TODO.md
   - name: Advanced topics
     items:
-    - name: Performance
-      href: Performance/PerfGettingStarted.md
     - name: Shared Experiences
       href: TODO.md
-    - name: Moving your project from HTK to MRTK
-      href: HTKToMRTKPortingGuide.md
+- name: How-to Guides
```
Comment (thalbern) [inline Documentation/toc.yml]: suggestion: the top level navigation already says guides / maybe just call this section How-tos or How to.... and then continue the children with eg setup global voice commands , trigger an action using MRTK button
that will make them shorter and we don't have the repetition of how to all the time?

File: Documentation/toc.yml [hunk 2]
```diff
@@ -97,20 +99,32 @@
...
-      href: Performance/PerfGettingStarted.md
     - name: Shared Experiences
       href: TODO.md
-    - name: Moving your project from HTK to MRTK
-      href: HTKToMRTKPortingGuide.md
+- name: How-to Guides 
+  items:
+  - name: How to setup global voice commands 
+    href: TODO.md
+  - name: How to setup a scene to be used for both HoloLens and VR with proper camera and scene content height
```
Comment (thalbern) [inline Documentation/toc.yml]: This is a bit long. Maybe just call it How to setup a scene for Mixed Reality?

**Comment by thalbern** [comment]
@Yoyozilla ignore if you know / do this already: you can run docfx locally - it just takes a couple of seconds. that way you can easier fix issues / iterate on issues in this PR without having to wait for the CI builder all the time :)

File: Documentation/toc.yml (hunk 2 above)
Comment (Yoyozilla) [reply Documentation/toc.yml]: good idea this is long for a place holder.

File: Documentation/toc.yml (hunk 1 above)
Comment (Yoyozilla) [reply Documentation/toc.yml]: done

File: Documentation/toc.yml [hunk 3]
```diff
@@ -96,20 +95,32 @@
...
       href: TODO.md
   - name: Advanced topics
     items:
-    - name: Performance
-      href: Performance/PerfGettingStarted.md
     - name: Shared Experiences
       href: TODO.md
-    - name: Moving your project from HTK to MRTK
-      href: HTKToMRTKPortingGuide.md
+- name: How-to
```
Comment (wiwei) [inline Documentation/toc.yml]: Unless someone is actually going to do this in the next few days, I recommend not adding new TODOs.

These are very frustrating to read (as a consumer) and are easy to forget about (as a producer)

Comment (keveleigh) [reply Documentation/toc.yml]: I'm going to go ahead and merge this PR as-is to get it in, then add this to my incoming docs PR.

**Pull request merged**